libcodechecker/libclient/client.py

```diff
@@ -254,9 +254,7 @@ def setup_client(product_url):
             # Only a "substring" match was found. We explicitly reject it
             # on the command-line!
             product_error_str = "It does not exist."
-        elif not product[0].connected:
-            product_error_str = "The database has issues, or the connection " \
-                                "is badly configured."
+
```

Comment:
This condition went missing. How will the client handle if a product has database issues, when, e.g. `CodeChecker cmd runs --url MyOfflineDatabase` is used?